pkg/tnf/interactive/spawner.go

```diff
@@ -272,6 +282,31 @@ func NewGoExpectSpawner() *GoExpectSpawner {
 	return &GoExpectSpawner{}
 }
 
+// LogStdErr logs stderr output to logger (warning)
+func logCmdMirrorPipe(cmdLine string, pipeToMirror io.Reader, name string, trace bool) io.Reader {
+	originalPipe := pipeToMirror
+	r, w, _ := os.Pipe()
+	tr := io.TeeReader(originalPipe, w)
+
+	go func() {
+		for {
+			buf := bufio.NewReader(tr)
+			line, _, err := buf.ReadLine()
+			if trace {
+				log.Trace(name + " for " + cmdLine + " : " + string(line))
+			} else {
+				log.Warn(name + " for " + cmdLine + " : " + string(line))
+			}
+			if err != nil {
```

Comment:
what's the rational behind logging either via Trace or Warn ?
I think logging everytime in trace level is good.  Also, you're always passing "true" and there's not a way to change that via command line, so it's really trace everytime.

Reply:
This was to address your comment about not having the stdout at warn level. leave stderr at warn and set stdout at trace